### PR TITLE
451 - Link & Node block list part 1

### PR DESCRIPTION
**Pull request by ahjyrkia**
closes https://github.com/HSLdevcom/jore-map-ui/issues/451

image

### CODE REVIEW

File: src/components/sidebar/routePathView/LinkNodeListViewTab.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                                    {link.id}
+                                </div>
+                            </div>
+                        </div>
+                        <div
+                            className={s.itemToggle}
+                        >
+                            <FaAngleRight />
+                        </div>
+                    </div>
```
Comment (Divinare) [inline src/components/sidebar/routePathView/LinkNodeListViewTab.tsx]: Duplicate HTML detected.

I would either make a list of objects [solmu, link, solmu, link, ...]
OR
renderItem(link);
renderItem(link.startNode);

And they would both use common method

File: src/components/sidebar/routePathView/RoutePathLinksTab.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                    timeAlignment &&
+                    <div className={s.timeAlignment}>
+                        Ajantasauspysäkki
+                    </div>
+                }
+            </>
+        );
+    }
+
+    public getItem = (itemType: string, link: IRoutePathLink) => {
```
Comment (ahjyrkia) [inline src/components/sidebar/routePathView/RoutePathLinksTab.tsx]: I'm not fond of this function. Improvements to this would be appreciated.

Comment (ahjyrkia) [reply src/components/sidebar/routePathView/RoutePathLinksTab.tsx]: Better solution is on its way

**Comment by Divinare** [comment]
rename NodeDescription as NodeTypeDescription?

**Pull request merged**